fusedrug/data/protein/tokenizer/build_protein_tokenizer_simple.py

```diff
@@ -111,8 +111,8 @@ def main(vocab_name: str, output_tokenizer_json_file: str, unknown_token: str):
 
     build_simple_vocab_protein_tokenizer(
         vocab=vocab_name,
-        unknown_token=unknown_token,  #'<UNK>',
-        save_to_json_file=output_tokenizer_json_file,
+        unknown_token=unknown_token,
+        save_to_json_file=output_tokenizer_json_file,  #'<UNK>',
```

Comment:
the unknown comment should be in line 114

Reply:
I don't know why this change was attributed to me - this isn't my code, and I've never even looked at it. It's not fully typed, so it won't pass mypy tests, so I can't even merge it properly...

Reply:
Changed it anyway. Weird.